### PR TITLE
Remove raster, rgdal, sp packages from dependencies

### PR DESCRIPTION
**Pull request by petrasovaa**
* removes packages rgdal, raster, sp
* replaced one line that was using raster to use terra instead, please check, I don't know if tests cover this
* landscapemetrics changed to 2.0.0
* added macOS dependency, CI was failing

### CODE REVIEW

File: R/pops_multirun.R
[the remark this answers is not in the document]
Comment (petrasovaa) [reply R/pops_multirun.R:1]: Are you sure? It looks like it's running on a list of numbers or something like it, not a raster. There is no terra in front of the other two which.min.

**Comment by petrasovaa** [comment]
Any idea why testthat runs forever sometimes in CI?

**Comment by ChrisJones687** [comment]
Not sure it occasionally gets hung up at loading packages and sometimes on
testthat. It doesn't get stuck locally on either my Mac or Windows machine
so I can't see why it would elsewhere.

File: R/pops_multirun.R
[the remark this answers is not in the document]
Comment (ChrisJones687) [reply R/pops_multirun.R:1]: Ignore my comment

**Pull request merged**